core/src/main/java/uk/gov/justice/services/core/handler/HandlerMethod.java

```diff
@@ -78,7 +78,16 @@ private static boolean isVoid(final Class<?> clazz) {
     }
 
     private static boolean isEnvelope(final Class<?> clazz) {
-        return JsonEnvelope.class.equals(clazz);
```

Comment:
Is it possible to use `Class.isAssignableFrom` ?

Reply:
No such method sadly according to my IDE

Reply:
Its an instance method; JsonEnvelope.class.isAssignableFrom(clazz) should work.